modules/core/src/loaders/xviz-websocket-loader.js

```diff
@@ -102,7 +103,12 @@ export default class XVIZWebsocketLoader extends XVIZLoaderInterface {
               message: message.data,
               onResult: this.onXVIZMessage,
               onError: this.onError,
-              debug: this._debug.bind('parse_message'),
+              debug: payload => {
+                this._debug.bind('parse_message');
```

Comment:
It was a bug in the existing code. Should be `debug: this._debug.bind(this, 'parse_message')`